app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -33,6 +33,17 @@ child :content_facet => :content_facet_attributes do
   node :remote_execution_by_default do
     Katello.remote_execution_by_default?
   end
+
+  user = User.current # current_user is not available here
+  child :permissions do
+    node(:view_lifecycle_environments) { user.can?("view_lifecycle_environments") }
+    node(:view_content_views) { user.can?("view_content_views") }
+    node(:promote_or_remove_content_views_to_environments) { user.can?("promote_or_remove_content_views_to_environments") }
+    node(:view_host_collections) { user.can?("view_host_collections") }
+    node(:create_job_invocations) { user.can?("create_job_invocations") }
+    node(:view_activation_keys) { user.can?("view_activation_keys") }
+    node(:view_products) { user.can?("view_products") }
+  end
```

Comment:
It seemed more correct to add Katello permissions to one of the facets (content facet or subscription facet) rather than add them in the Foreman codebase, but of the two options (content facet or subscription facet) I wasn't sure which is more relevant. I went with content facet.

This also includes `create_job_invocations` which technically shouldn't even be in Katello at all, as it's from the foreman_remote_execution plugin. But it seems silly to make another plugin extend the host rabl in the same way, so I just did it here.